### PR TITLE
chore(dashmate): temporary disable HPMN registration with mobile wallets

### PR DESCRIPTION
**Pull request by shumkov**
## Issue being fixed or feature implemented

Evonodes registration is not implemented yet in Mobile Wallets

## What was done?

- Temporary hide options to register masternode with mobile wallets in `dashmate setup` 

## How Has This Been Tested?

None

## Breaking Changes

None

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: packages/dashmate/src/listr/tasks/setup/regular/registerMasternodeGuideTaskFactory.js [hunk 1]
```diff
@@ -62,8 +63,8 @@ function registerMasternodeGuideTaskFactory() {
               message: 'Which wallet will you use to store keys for your masternode?',
               choices: [
                 { name: REGISTRARS.CORE, message: 'Dash Core Wallet' },
-                { name: REGISTRARS.ANDROID, message: 'Android Dash Wallet' },
-                { name: REGISTRARS.IOS, message: 'iOS Dash Wallet' },
+                // { name: REGISTRARS.ANDROID, message: 'Android Dash Wallet' },
+                // { name: REGISTRARS.IOS, message: 'iOS Dash Wallet' },
                 { name: REGISTRARS.OTHER, message: 'Other' },
```
Comment (pshenmic) [inline packages/dashmate/src/listr/tasks/setup/regular/registerMasternodeGuideTaskFactory.js:68]: Are there any differences between `Dash Core Wallet` and `Other` register methods?

Comment (shumkov) [reply packages/dashmate/src/listr/tasks/setup/regular/registerMasternodeGuideTaskFactory.js:68]: Nope :)

Comment (pshenmic) [reply packages/dashmate/src/listr/tasks/setup/regular/registerMasternodeGuideTaskFactory.js:68]: Yeah, looks like. We can always enable it back, when we implement something

Comment (shumkov) [reply packages/dashmate/src/listr/tasks/setup/regular/registerMasternodeGuideTaskFactory.js:68]: It seems DMT integration is on going. I wouldn't disable prompt atm to enable it back in a week.

**Review by pshenmic — APPROVED** [review]

**Comment by shumkov** [comment]
Functionality is already released

**Pull request closed without merging**